src/config/chain-workarounds.ts

```diff
@@ -3,7 +3,6 @@ import { ETHEREUM_NETWORK } from './networks/network.d'
 // How much to add to gasLimit per network
 // Defaults to 1 (i.e. no extra gas)
 export const EXTRA_GAS_FACTOR = {
-  [ETHEREUM_NETWORK.ARBITRUM]: 1.2, // +20%
```

Comment:
This was added because of NFTs on Arbitrum. They can be now sent w/o a problem with no extra gas.